admin/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copy template files into <build_dir>/horace_config
+set(WORKER_CONFIG_DIR "${CMAKE_BINARY_DIR}/horace_config")
+file(MAKE_DIRECTORY ${WORKER_CONFIG_DIR})
+
+# Format and copy horace_on template
+file(READ "horace_on.m.template" _horace_on_template)
+string(REPLACE
+    "default_horace_path ='/usr/local/mprogs/Horace/horace_core';"
```

Comment:
these explicit replacements are very fragile; if the template is ever updated this will fail; can you replace it with a "replace `default_horace_path =*`" match?